src/Resources/config/services.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="Sylius\AdminOrderCreationPlugin\Controller\SelectNewOrderCustomerAction">
+            <argument type="service" id="form.factory" />
+            <argument type="service" id="templating" />
+        </service>
+
+        <service id="Sylius\AdminOrderCreationPlugin\Form\Type\NewOrderCustomerSelectType">
+            <tag name="form.type" />
+        </service>
+        <service id="Sylius\AdminOrderCreationPlugin\Form\Type\NewOrderType">
+            <argument>%sylius.model.order.class%</argument>
+            <argument>%sylius.form.type.order.validation_groups%</argument>
+            <tag name="form.type" />
+        </service>
+        <service id="Sylius\AdminOrderCreationPlugin\Form\Type\OrderItemType">
+            <argument>%sylius.model.order_item.class%</argument>
+            <argument>%sylius.form.type.order_item.validation_groups%</argument>
+            <argument type="service" id="sylius.form.data_mapper.order_item_quantity" />
+            <tag name="form.type" />
+        </service>
+        <service id="Sylius\AdminOrderCreationPlugin\Form\Type\ShipmentType">
+            <argument>%sylius.model.shipment.class%</argument>
+            <argument>%sylius.form.type.shipment.validation_groups%</argument>
+            <tag name="form.type" />
+        </service>
+        <service id="Sylius\AdminOrderCreationPlugin\Form\Type\PaymentType">
+            <argument>%sylius.model.payment.class%</argument>
+            <argument>%sylius.form.type.payment.validation_groups%</argument>
+            <tag name="form.type" />
+        </service>
+
+        <service id="Sylius\AdminOrderCreationPlugin\Factory\OrderFactory" decorates="sylius.factory.order" public="false">
```

Comment:
I guess it will crash the application, as it's decorating order factory with the one implementing this plugin's `OrderFactoryInterface` whereas someone might rely on it implementing any other interface.

I'd vote for making this factory a standalone service, but I'm not sure if ResourceBundle supports that - if not, I'll be happy to merge a PR with that to the `1.2` branch to have it released by the end of May, it's crucial to our plugins ecosystem.